### DISCUSSION TITLE
Compile problem

### DISCUSSION THREAD

**Discussion by Flower-Wang1024** [bitwuzla/bitwuzla#107] [Q&A]
hello, I'm a student. I want to use bitwuzla but I encountered problem when compling. I tried to solve it but failed. Can you suggest a solution, thanks!
![image](https://github.com/bitwuzla/bitwuzla/assets/62416914/1917ff94-e4f6-46fa-9413-8f147750e890)

**Comment by mpreiner** [comment]
Can you post the configure output?

**Reply by Flower-Wang1024** [reply]
Hello, now I want to use python-bindings, but another problem arises, how can I fix it? thanks!
![image](https://github.com/bitwuzla/bitwuzla/assets/62416914/add23999-88a6-4528-9198-1858de0e622f)

**Reply by mpreiner** [reply]
Can you post the full log? Without it it's hard to tell what's going on.

**Reply by aytey** [reply]
Something I saw is that Meson is trying to compile with `cl.exe` when doing `pip install` ...
